vtds_cluster_kvm/private/scripts/deploy_cluster_to_blade.py

```diff
@@ -464,6 +464,13 @@ def __add_etc_hosts_entry(self):
         with open("/etc/hosts", mode='w', encoding="UTF-8") as hosts:
             hosts.writelines(hosts_lines)
 
```

Comment:
REVIEWERS: we don't want to wait for a network booted compute node if it is not expected to be powered on at the end of cluster creation. This function allows the code that normally waits for nodes to boot to distinguish this case.